lint/lint.py

```diff
@@ -163,7 +163,7 @@ def check_regexp_line(repo_root, path, f):
     return errors
 
 def check_parsed(repo_root, path, f):
-    source_file = SourceFile(repo_root, path, "/")
+    source_file = SourceFile(repo_root, path, "/", contents=f.read())
```

Comment:
That would mean you could only call `SourceFile.open()` once, I think, and would make it less consistent with the other cases.